### ISSUE TITLE
Add assert when 'fixed elastic time step' is smaller than 'max computational time step'

### ISSUE THREAD

**Issue by EstherHeck** [geodynamics/aspect#3843]
As it does not make sense to use a 'fixed elastic time step' that is smaller than 'max computational time step' and also produces unrealistic model results (velocities go crazy), aspect should warn users, when they accidentally try to do this

**Comment by anne-glerum** [comment]
Thanks for the suggestion! Do you mean `Maximum time step`? Its default is 5.69e+300, so when users don't set it to something different, this will fail. Also, should the `Fixed elastic time step` be larger than the `Maximum time step` or than the actual current timestep?

**Comment by EstherHeck** [comment]
yes I mean 'Maximum time step'. I think it can be equal as well. Models don't fail in this case, they just oscillate.
